process/sync/baseForkDetector.go

```diff
@@ -470,3 +495,32 @@ func (bfd *baseForkDetector) shouldAddBlockInForkDetector(
 
 	return nil
 }
+
+func (bfd *baseForkDetector) computeForcedFork(
+	header data.HeaderHandler,
+	state process.BlockHeaderState) {
```

Comment:
add enter after State,

Reply:
done

Reply:
not done. enter after State put ") {" in new line

Reply:
done